src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.List;
 import java.util.NoSuchElementException;
+import java.util.Optional;
```

Comment:
Consider removing the unused import `java.util.NoSuchElementException`, as it is not used in the code. Keeping the code clean and free of unused imports helps to maintain readability and avoid confusion. 🧹

Reply:
@Override
    public Driver get(Long id) {
        return driverDao.get(id).orElseThrow(() ->
            new NoSuchElementException("Can't get driver by id: " + id)
        );
    }
    
    But this exception exists in method